File: esercizi_python/compito_python_3/Tennis.py
from Campo import Campo


class Tennis(Campo):
    def __init__(self, cod, nome, larghezza, lunghezza, costorario, temperatura, terreno):
        super().__init__(cod, nome, larghezza, lunghezza, costorario)
        self.temperatura = temperatura
        self.terreno = terreno

    def toString(self):
        return "tennis\t"+super().toString()+str(self.temperatura)+"\t"+self.terreno+"\t-\t-\t"+str(self.costorario)

File: esercizi_java/NetBeansProjects/Circolo_Sportivo/src/circolo_sportivo/Tennis.java
package circolo_sportivo;

/**
 *
 * @author tobiasacchetto
 */
public class Tennis extends Campo{
    private String terreno;
    private float temperatura;
    
    public Tennis(int cod,String nome,float larghezza, float lunghezza, float costorario, float temperatura, String terreno){
        super(cod,nome,larghezza,lunghezza,costorario);
        this.temperatura= temperatura;
        this.terreno=terreno;
    }
    
    public String toString(){
        return "tennis\t"+super.toString()+temperatura+"\t"+terreno+"\t-\t-\t"+super.costorario;
    }
    
}
